README.md

```diff
@@ -14,17 +14,10 @@
 
 ## Table of Content
 
-- [Getting Started](#getting-started)
-- [Usage](#usage)
-- [Documentation](#documentation)
-  - [`AuthProvider`](#authprovider)
-  - [`useAuth`](#useauth)
-    - [`login`](#login)
-    - [`logout`](#logout)
-    - [`handleAuth`](#handleauth)
-    - [`isAuthenticated`](#isauthenticated)
-    - [`user`](#user)
-    - [`authResult`](#authresult)
+- [Table of Content](#table-of-content)
```

Comment:
If you look at the rich output of this markdown, the Table of Content does not render correctly.